modules/ossm-migrating-multitenant-migration-guide.adoc

```diff
@@ -0,0 +1,299 @@
+
+*2.* Add your `tenant` label to each one of your dataplane namespaces.
+
+   With 2.6, we enrolled namespaces into the mesh by adding them to the Service Mesh Member Roll resource. In 3.0, you must label each one of your dataplane namespaces with this label. For every namespace in your Service Mesh Member Roll, add your tenant label to the namespace.
+
+   ```sh
+   oc label ns bookinfo tenant=tenant-a
+   ```
+
+   Now we are ready to migrate our workloads from our 2.6 controlplane to our 3.0 controlplane.
+
+=== Migrate Workloads
+
+1. Update injection labels on the dataplane namespace.
+
+   Here we're adding two labels to the namespace:
+
+   * The `istio.io/rev: istio-tenant-a` label which ensures that any new pods that get created in that namespace will connect to the 3.0 proxy.
+   * The `maistra.io/ignore-namespace: "true"` label which will disable sidecar injection for 2.6 proxies in the namespace. This ensures that 2.6 will stop injecting proxies in this namespace and any new proxies will be injected by the 3.0 controlplane. Without this, the 2.6 injection webhook will try to inject the pod and it will connect to the 2.6 proxy as well as refuse to start since it will have the 2.6 cni annotation.
+
+   **Note:** that once you apply the `maistra.io/ignore-namespace` label, any new pod that gets created in the namespace will be connected to the 3.0 proxy. Workloads will still be able to communicate with each other though regardless of which controlplane they are connected to.
+
+   ```sh
+   oc label ns bookinfo istio.io/rev=istio-tenant-a maistra.io/ignore-namespace="true" --overwrite=true
+   ```
+
+*2*. `curl` the productpage pod in `bookinfo` to ensure proxies can still communicate with one another.
+
+   ```sh
+   oc exec -it -n bookinfo deployments/productpage-v1 -c istio-proxy -- curl localhost:9080/productpage
+   ```
+
+   You should see
+
+   ```html
+   ...
+           <p>Absolutely fun and entertaining. The play lacks thematic depth when compared to other plays by Shakespeare.</p>
+           <small>Reviewer2</small>
+
+
+           <font color="black">
+             <!-- full stars: -->
+
+             <span class="glyphicon glyphicon-star"></span>
+
+             <span class="glyphicon glyphicon-star"></span>
+
+             <span class="glyphicon glyphicon-star"></span>
+
+             <span class="glyphicon glyphicon-star"></span>
+
+             <!-- empty stars: -->
+
+             <span class="glyphicon glyphicon-star-empty"></span>
+
+           </font>
+
+
+         </blockquote>
+
+         <dl>
+           <dt>Reviews served by:</dt>
+           <u>reviews-v2-6dd458b5db-frrlb</u>
+
+         </dl>
+   ...
+   ```
```

Comment:
Same comment as below with removing this example since it is specific to bookinfo and not generally useful to users' actual apps.